docs/ops.html

```diff
@@ -686,7 +686,7 @@
 
   <h3><a id="monitoring" href="#monitoring">6.6 Monitoring</a></h3>
 
-  Kafka uses Yammer Metrics for metrics reporting in both the server and the client. This can be configured to report stats using pluggable stats reporters to hook up to your monitoring system.
+  Kafka uses Yammer Metrics for metrics reporting in the server and a custom metrics registry in the client. Both expose metrics via JMX and can be configured to report stats using pluggable stats reporters to hook up to your monitoring system.
```

Comment:
I assume that Kafka Metrics were introduced in order to minimize dependencies that the client pulls in. If that's true, I can make that more clear here:

> and a custom metrics registry in the client to minimize transitive dependencies pulled in to client applications